src/pk_compiler.c

```diff
@@ -991,6 +1000,9 @@ static bool matchAssignment(Compiler* compiler) {
   if (match(compiler, TK_ANDEQ)) return true;
   if (match(compiler, TK_OREQ)) return true;
   if (match(compiler, TK_XOREQ)) return true;
+  if (match(compiler, TK_MODEQ)) return true;
```

Comment:
`TK_MODEQ` already handled above (after the `TK_DIVEQ`), remove it here.

Reply:
Both done.